applications/CompressiblePotentialFlowApplication/python_scripts/apply_far_field_process.py

```diff
@@ -24,20 +29,32 @@ def __init__(self, Model, settings ):
         settings.ValidateAndAssignDefaults(default_parameters);
 
         self.model_part = Model[settings["model_part_name"].GetString()]
-        self.velocity_infinity = KratosMultiphysics.Vector(3)#array('d', [1.0, 2.0, 3.14])#np.array([0,0,0])#np.zeros(3)#vector(3)
-        self.velocity_infinity[0] = settings["velocity_infinity"][0].GetDouble()
-        self.velocity_infinity[1] = settings["velocity_infinity"][1].GetDouble()
-        self.velocity_infinity[2] = settings["velocity_infinity"][2].GetDouble()
-        #self.density_infinity = settings["density_infinity"].GetDouble() #TODO: must read this from the properties
+        self.fluid_model_part = self.model_part.GetRootModelPart()
+
+        self.angle_of_attack = settings["angle_of_attack"].GetDouble()
+        self.mach_inf = settings["mach_infinity"].GetDouble()
+        self.density_inf = settings["density_infinity"].GetDouble()
+        self.seed_of_sound_inf = settings["speed_of_sound"].GetDouble()
+        self.heat_capacity_ratio = settings["heat_capacity_ratio"].GetDouble()
         self.inlet_phi = settings["inlet_phi"].GetDouble()
-        self.model_part.ProcessInfo.SetValue(CompressiblePotentialFlowApplication.VELOCITY_INFINITY,self.velocity_infinity)
 
+        # Computing free stream velocity
+        self.u_inf = self.mach_inf * self.seed_of_sound_inf
+        self.velocity_infinity = KratosMultiphysics.Vector(3)
+        self.velocity_infinity[0] = round(self.u_inf*math.cos(self.angle_of_attack),8)
```

Comment:
```suggestion
        self.velocity_infinity[0] = self.u_inf*math.cos(self.angle_of_attack)
```
why round?

Reply:
I observed that math.cos(0) does not deliver 1.0 but 1.00000000001. This was breaking a test since we are using very low tolerances and I found this solution but I am open to ideas.